java/code/src/com/redhat/rhn/taskomatic/task/RepoSyncTask.java

```diff
@@ -62,53 +63,36 @@ public String getConfigNamespace() {
     public void execute(JobExecutionContext context) {
         List<Long> channelIds = getChannelIds(context.getJobDetail().getJobDataMap());
 
-        String[] lparams = {"no-errata", "latest", "sync-kickstart", "fail"};
+        List<String> lparams = Arrays.asList("no-errata", "latest", "sync-kickstart", "fail");
         List<String> ltrue = Arrays.asList("true", "1");
-        List<String> params = new ArrayList<>();
-        for (String p : lparams) {
-            if (context.getJobDetail().getJobDataMap().containsKey(p)) {
-                if (ltrue.contains(context.getJobDetail().getJobDataMap().get(p).toString()
-                        .toLowerCase().trim())) {
-                    params.add("--" + p);
-                }
-            }
-        }
+        List<String> params = lparams.stream()
+                .filter(p -> context.getJobDetail().getJobDataMap().containsKey(p) &&
+                    ltrue.contains(context.getJobDetail().getJobDataMap().get(p).toString().toLowerCase().trim()))
+                .map(p -> "--" + p)
+                .collect(Collectors.toList());
 
         for (Long channelId : channelIds) {
             Channel channel = ChannelFactory.lookupById(channelId);
             if (channel != null) {
                 log.info("Syncing repos for channel: {}", channel.getName());
 
                 try {
-                    executeExtCmd(getSyncCommand(channel, params).toArray(new String[0]));
+                    executeExtCmd(getSyncCommand(channel, params).toArray(new String[0]), true);
```

Comment:
I'm probably blind on this one. But If we call it with true, I think it will create the input stream and store the logs. Don't you want to make it the other way around?

Reply:
You're right! I inverted the logic in the process and forget to apply it everywhere. Nice catch!

Reply:
`executeExtCmd()` added parameter is `noStdoutLog`, so the logic is OK here